uuid.js

```diff
@@ -59,6 +59,16 @@
     _hexToByte[_byteToHex[i]] = i;
   }
 
+  // Force _hexToByte to be changed to fast-property object in V8
+  (function toFastProperties(obj) {
+    /*jshint -W027*/
+    function f() {}
+    f.prototype = obj;
+    return f;
+    eval(obj);
```

Comment:
Yes, it's unreachable and noop, but it prevents V8 from removing dead code. 
Kind of voodoo.